coderedcms/templates/coderedcms/blocks/accordion_block.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'coderedcms/blocks/base_block.html' %}
+
+{% load wagtailcore_tags coderedcms_tags %}
+
+{% block block_render %}
+
+{% with a_id=self.accordion.id %}
+
+<button class="btn {{self.button_size}} {{self.button_style}}" type="button" data-toggle="collapse" data-target="#accordion-{{a_id}}" aria-expanded="false" aria-controls="accordion-{{a_id}}">{{self.button_title}}</button>
```

Comment:
This template should look and function exactly like the bootstrap example: https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example

![image](https://user-images.githubusercontent.com/13453401/181801996-3388f435-6540-4e61-b347-bba1de54564c.png)